int32[] map_landmarks
---
bool plotted
